README.md

```diff
@@ -8,7 +8,11 @@
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com)
 [![Join chat](https://img.shields.io/badge/Mattermost-join_chat-brightgreen.svg)](https://chat.mycroft.ai)
 
-# Mycroft
+# This project is no longer actively maintained
+
+Mycroft core is no longer maintaiend and probably likely not work on your computer anymore. [Open Voice OS](https://github.com/OpenVoiceOS/) and [Neon-core](https://github.com/NeonGeckoCom/NeonCore) are both spiritual successors to Mycroft. (And some of the old code may live on there.)
```

Comment:
please link https://openvoiceos.org instead of our github :)

OVOS is the direct continuation of the mycroft code base, so a lot of the old code does live there! and probably will for a long long time. but i guess the current wording makes the readme more future proof

Reply:
Done